bin/deploy

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+set -e
+
+echo 'Setting env...'
+export MIX_ENV=prod
+
+echo 'Copying config from server...'
+sshpass -p $PASSWORD scp -o StrictHostKeyChecking=no ben@$IP:/home/utilshare-config/prod.secret.exs ./config/prod.secret.exs
```

Comment:
This is what I stored the config on the server for -- on my microblog app. Did you do it with env vars?

Reply:
https://github.com/Devlus/microblog/blob/master/buildDep

didn't do anything with the config.
Distillery bundled it up I think.

Reply:
Did you run that locally or on your CI server? I store it on the server so that I can copy it to travis and deploy on their server. Not necessary if you build locally

Reply:
I was building on my local machine, and and untarring the release on the server. But if this way works with Continuous Deployments then dope, merge it.